### PR TITLE
feat: shortcuts display bar

### PR DESCRIPTION
**Pull request by Himanshu-Singh-Chauhan**
#### UPDATE
made it scrollable
Code_37vzH50SGZ

works with trackpad and mouse.
looks janky in gif, but its smooth with trackpad.

---
context - 
we need to show shortcuts to users, and we are not able to put these shortcuts in title bar as we require, because we are limited by vscode api.
so, we put this shortcuts bar for the time being until user gets comfortable with shortcuts.
[Discord msg by Pan](https://discord.com/channels/1237009981803462729/1255013588473610250/1260998423151116328)

what changed - 
- add new bar at top to display shortcuts.
image
- tried to make it compact. styles can be tweaked.
- also gives us few pixels of extra real estate overall, might not be noticeable.

### CODE REVIEW

File: extensions/vscode/package.json [hunk 1]
```diff
@@ -421,12 +421,12 @@
...
           "command": "pearai.winResizeAuxiliaryBarWidth",
           "group": "navigation@0",
-          "when": "view == pearai.continueGUIView && (isWindows || isLinux)"
+          "when": "false && view == pearai.continueGUIView && (isWindows || isLinux)"
         },
         {
           "command": "pearai.macResizeAuxiliaryBarWidth",
           "group": "navigation@0",
-          "when": "view == pearai.continueGUIView && isMac"
+          "when": "false && view == pearai.continueGUIView && isMac"
```
Comment (Himanshu-Singh-Chauhan) [inline extensions/vscode/package.json:429]: for now, we disable it using `false` and later we remove it if we are satisfied with the shortcuts bar.

**Review by Fryingpannn — COMMENTED** [review]
I made some changes, now it looks like below. I think we can do one word descriptions, and remove "+"s and separators.
image

Some changes needed:
- Hovering over button makes all black. I think we can remove the hovers altogether. No hover effects
Screenshot 2024-07-14 at 3 19 53 PM

- When I click on the `CMD+;` button, it closes the chat panel and opens this page. Seems like a bug.
- We can also add `Back CMD+0` shortcut to the bar
image
- We can remove the 'Big chat - CMD+[' in the top bar now since we have it in this new shortcuts row.

**Comment by Himanshu-Singh-Chauhan** [comment]
pushed new changes as requested:
- one word descriptions, and remove "+"s and separators.
- No hover effects
- fix - CMD+; bug
- add CMD+0 shortcut
- remove - Big chat - CMD+[ - if you don't see this removed, might have to recompile or reload properly.

for now 3 shortcuts fits perfectly, but if add more then we make shortcuts row scrollable.

**Comment by Himanshu-Singh-Chauhan** [comment]
made it scrollable
Code_37vzH50SGZ

works with trackpad and mouse.
looks janky in gif, but its smooth with trackpad.

**Comment by nang-dev** [comment]
Yoo this is nice, but maybe worth having a divider? Like a  "|"

**Review by Fryingpannn — APPROVED** [review]

**Comment by Himanshu-Singh-Chauhan** [comment]
image
borders

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**